string_manipulation.rb

```diff
@@ -75,6 +82,9 @@ def reverse_words(my_words)
   return my_words
```

Comment:
Similar to the first method, this method also updates the characters in place and hence, you don't need to return my_words.